lib/aes_256_cbc_encryptor.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class Aes256CbcEncryptor
+  def initialize(hex_secret, hex_iv)
+    @secret = [hex_secret].pack('H*')
+    @iv = [hex_iv].pack('H*')
+    # Pad with zero bytes to correct length
+    @secret << ("\x00" * (32 - secret.length))
```

Comment:
👀 shouldn't we error if key length wasn't adequate?

Reply:
Slight preference for error on startup. Like you said, prevents from shipping into production in the first place.

Reply:
where did you apply the change?